content/en/platform/corda/5.0/developing-applications/api/api-crypto.md

```diff
@@ -181,16 +179,11 @@ public final class CompositeKeyNodeAndWeight {
         return otherKey.weight == weight;
     }
 }
-
 ```
 
 ## CordaOID.java
 
-{{< tabs name="CordaOID">}}
-{{% tab name="Kotlin" %}}
```

Comment:
where is the removal of the `{{< tabs ...` and `{{% tab ...` enclosures ?

Reply:
There was no enclosures, as this content is still in draft mode and I originally thought there would be Kotlin and java  I had tabs in as a place stopper but since it's just java right now I removed it!